### DISCUSSION TITLE
Moving license to another project

### DISCUSSION THREAD

**Discussion by resimone** [OFFLINE-GmbH/oc-boxes-support#41] [Help] [answered]
Is that possibile?
On the same October CMS user area
Thanks in advance!

**Comment by tobias-kuendig** [comment]
You can unbind a license from a project by clicking the "x" beside the plugin in a project's detail view.

But to be honest, I am not sure what happens to the license afterwards. If you are unsure, I suggest you contact the site admins at hello@octobercms.com or post this question in Slack.

**Comment by resimone** [comment, marked as answer]
Hello Tobias,
First of all thanks for your kind reply.
Your plugin is really great, thanks for your work
We’re going to customize it a little for an our customer but just as it is right now it’s great!

Have a nice day 

SIMONE RESCA
tecnotrade | TECNOLOGIA & STRATEGIA
 
E ***@***.*** ***@***.***> | W https://tecnotrade.com <http://www.tecnotrade.com/> | T 051 0019916 | M 348.2616422 

Avvertenza: la presente casella e-mail ed i messaggi da essa derivanti, sono di esclusivo utilizzo aziendale/lavorativo e mai personale. Risposte al presente messaggio: si avvisa il destinatario che eventuali sue risposte potranno essere lette dall’intera azienda/ufficio/reparto di appartenenza del mittente. Se per un disguido avete ricevuto questa e-mail senza esserne i destinatari vogliate cortesemente distruggerla e darne informazione all’indirizzo mittente inviando una mail a ***@***.*** ***@***.***>
